src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -2784,6 +2783,56 @@ static irqreturn_t sched_cq_isr(int irq, void *arg)
 	return IRQ_HANDLED;
 }
 
+inline void init_exec(struct sched_exec_core *exec_core)
+{
+	unsigned int i;
+        exec_core->scheduler = &g_sched0;
```

Comment:
Use 'tab' at the start of the line.

Reply:
No. All these line are using space. Linux driver required 'tab'.

Reply:
fixed